tardis/plasma/standard_plasmas.py

```diff
@@ -28,6 +29,7 @@
     adiabatic_cooling_properties,
     two_photon_properties,
     isotope_properties,
+    nlte_properties_new,
```

Comment:
Why the `_new` in the name?

Reply:
it was just a temporary solution by me and Christian. We just need a way of differentiating between this and numerical nlte that already exists in the code.

Reply:
this is confusing. We should find a good workaround

Reply:
I can name it `nlte_solver_properties`